crc-bundle-info.json.sample

```diff
@@ -70,6 +70,16 @@
 	"size": "9129426944"
 	"sha256sum": "49766122a0834d62c1a24fb4e0de30cd7a39b8112083aa5e01fc26f16c15aed3"
       }
+    ],
+    "additionalFiles": [
```

Comment:
would it be possible to put all files here instead of just additional ? Maybe as a follow-up.  I understand in shell, it will be real pain to write :/

Reply:
The other files are already referenced in other places so I felt less need for it
```
    # Name of a file containing an SSH private key which can be used to connect to
    # the cluster nodes
    "sshPrivateKeyFile": "id_ecdsa_crc",
    # Name of the kubeconfig file stored in the bundle
    "kubeConfig": "kubeconfig",
    # Name of the file containing the kubeadmin password for use in the
    # openshift console
    "kubeadminPasswordFile": "kubeadmin-password"
    # pull secret that can be used to fetch OpenShift container images (optional)
    # "openshiftPullSecret": "default-pull-secret"
[...]
      # Disk image used by the node, the 'storage' object will contain more
      # details about its format
      "diskImage": "crc.qcow2"
```

Adding them to 'additionalFiles' would be redundant, but it would also allow us to give checksums and various metadata about these files , which can be a good thing. We actually already have the same redundancy with the qcow2 image, `"diskImage": "crc.qcow2"` is a key into the `"diskImages"` array which gives more details about the actual image.

So yeah, we could add more files in there, but I'll keep that as a future improvement :)
Maybe I should use a different name than `additionalFiles` though if we want to use it more widely.

Reply:
I renamed the key to the more generic `fileList`